symfony/plugins/orangehrmTimePlugin/Dao/TimesheetDao.php

```diff
@@ -970,4 +970,26 @@ public function saveAndUpdateTimesheetItems(array $timesheetItems): void
         }
         $this->getEntityManager()->flush();
     }
+
+    /**
+     * @param  int  $timesheetId
+     * @param  int  $activityId
+     * @param  int  $projectId
+     * @return bool
+     */
+    public function isDuplicateTimesheetItem(
+        int $timesheetId,
+        int $activityId,
+        int $projectId
+    ): bool {
+        $qb = $this->createQueryBuilder(TimesheetItem::class, 'timesheetItem');
+        $qb->andWhere('timesheetItem.timesheet = :timesheetId');
+        $qb->setParameter('timesheetId', $timesheetId);
+        $qb->andWhere('timesheetItem.project = :projectId');
+        $qb->setParameter('projectId', $projectId);
+        $qb->andwhere('timesheetItem.projectActivity = :activityId');
+        $qb->setParameter('activityId', $activityId);
+
+        return $this->getPaginator($qb)->count() === 0;
```

Comment:
```suggestion
        return $this->getPaginator($qb)->count() > 0;
```
I feel according to the method name `isDuplicateTimesheetItem` logic should change like this. What do you think?

Reply:
greater than 0 means there can be 1 or more neh?? then we are assuming that there might be already a duplicate record